README.md

```diff
@@ -20,6 +20,25 @@ convert the map files to alternate formats.  The JSON website describes
 the file format and includes links to parsing libraries in many
 programming languages.
 
+For this repository, we're formatting JSON files with each entry on its
+own line, and 2 spaces for indentation.  You can get this formatting using
+either `jq` or `python`:
+
+- jq: `jq --indent 2 --ascii-output . mapname.nv.json`
```

Comment:
I'm leaning toward supporting embedded platforms with limited resources and unable to process UTF8.  7-bit ASCII is a subset of UTF8, and the file is still valid UTF8.

This came up when I was using the High Speed JSON file for testing, and my `jq` and Python output differed on that one byte.

Also notable that I'm trying to encode a specific 8-bit byte on High Speed, and not any specific letter/character (in this case, "Ä").  If we UTF8-encode the JSON, that 0xC4 byte is actually written as 0xC3 0x84.

Reply:
yes that field is pita, might be better to just encode it as bytes as there is no encoding type that is proper for it

Reply:
Well, that character shows up in the "default" attribute for the field, so changing it to `bytes` from `ch` won't help.  I guess I could just remove the default value since it's of limited use in this context.

Reply:
you could leave it with a double escape :-)